lib/marshalls/repo.marshall.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const BaseMarshall = require('./baseMarshall')
+const urlExists = require('url-exists')
+const MARSHALL_NAME = 'repo'
+
+class Marshall extends BaseMarshall {
+  constructor (options) {
+    super(options)
+    this.name = MARSHALL_NAME
+  }
+
+  title () {
+    return 'Identifying package repository...'
+  }
+
+  validate (pkg) {
+    return this.packageRepoUtils.getPackageInfo(pkg.packageName).then(data => {
+      const lastVersionData = data.versions && data.versions[data['dist-tags'].latest]
```

Comment:
what about instead of looking inside the latest package, or if it doesn't exist there, we also look at the top level `repository` field?

Reply:
by the way, do we have another marshall that checks the `homepage` top level entry? if not maybe we can also check this here

Reply:
I think it should be done in a separate marshall, but I like the idea :)

Reply:
I'd leave this check here, but it's a good question about amount of marshalls we can have. As we add them, it means a task for each one and it could be un-pleasing on the terminal interface for the user.

This adds another idea - maybe we can add a silent switch that we can control with an env var?
You're welcome to add issues on the above :-)

Reply:
yeah, the number of marshalls is a limited resource. Will fix then :)